module/index.js

```diff
@@ -106,6 +106,7 @@ import toLowerCase from './toLowerCase';
 import toUpperCase from './toUpperCase';
 import uncurry from './uncurry';
 import uncurry3 from './uncurry3';
+import unfold from './unfold';
```

Comment:
yeah makes sense, just feels a bit like "cheating" since we would have to do `ƒ(...){return...}` :disappointed: